configs/components/nssm.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
You also need to change bin/build-windows.rb. Windows bulids are currently broken.